adoc/headers/multipointerlegacy.h

```diff
@@ -61,6 +61,10 @@ class [[deprecated]] multi_ptr<ElementType, Space, access::decorated::legacy> {
   // Returns the underlying OpenCL C pointer
   pointer_t get() const;
 
+  std::add_pointer_t<value_type> get_raw() const;
```

Comment:
Should this have used `ElementType` instead of `value_type`? `value_type` doesn't seem to be a thing in this specialization.

Reply:
Yeah good catch, the legacy specialization doesn't have `value_type` so this should be either `element_type` or `ElementType`.

I've created a follow-up PR to address this: https://github.com/KhronosGroup/SYCL-Docs/pull/437.

Reply:
In second thought, I've decided to introduce `value_type` to the legacy specialization rather than changing `get_raw`, so that the `legacy` and non-`legacy` specializations are closer together.